api-client/src/main/java/com/xing/api/RealCallSpec.java

```diff
@@ -142,6 +144,16 @@ public void onResponse(Call call, okhttp3.Response rawResponse) {
     }
 
     @Override
+    public Single<RT> singleStream() {
+        return stream().toSingle();
+    }
+
+    @Override
+    public Completable completableStream() {
+        return stream().toCompletable();
```

Comment:
I'm not sure it should be that easy... the problem with `Completable` is that its ignoring the response... but by default `CallSpec` is designed to expect one.